src/components/nav-bar/nav-bar.tsx

```diff
@@ -0,0 +1,17 @@
+import { Component, Host, h, Prop } from "@stencil/core";
+
+@Component({
+  tag: "oui-nav-bar",
+  styleUrl: "nav-bar.css"
+})
+export class NavBar {
+  @Prop() Dark: boolean;
```

Comment:
Props should be lowercase.

Reply:
Yeah, the Dark prop there wasn't really necessary, will remove.